.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+This page contains the contributing guidelines.
+
+## Setup
+### prerequisite
+- Node.js installed. 
```

Comment:
I think I might have used code that requires at least Node 8. I would recommend people to use the LTS version for development at this point.

Reply:
v8.0.0 it is then ?

Reply:
Sure!